testApi/src/main/kotlin/testApi/context/MockContext.kt

```diff
@@ -40,6 +40,12 @@ class MockContext(
 
     override val platforms: Map<PlatformData, EnvironmentAndFacade>
         get() = testPlatforms ?: throw IllegalStateException("This mock context doesn't provide platforms data")
+    override val pointsUsed: MutableSet<ExtensionPoint<*>>
+        get() = TODO("not implemented") //To change initializer of created properties use File | Settings | File Templates.
+    override val pointsPopulated: MutableSet<ExtensionPoint<*>>
+        get() = TODO("not implemented") //To change initializer of created properties use File | Settings | File Templates.
+    override val unusedPoints: Collection<ExtensionPoint<*>>
+        get() = TODO("not implemented") //To change initializer of created properties use File | Settings | File Templates.
```

Comment:
Do not leave autogenerated methods stubs. If you are sure you do not want to provide any implementation throw explicit exception

Reply:
Changed, now unusedPoints are mockable.